data-base/db.go

```diff
@@ -2,9 +2,11 @@ package database
 
 import (
 	"main/repo"
+	"sort"
 
 	"github.com/jmoiron/sqlx"
-	_ "github.com/lib/pq"
+	_ "github.com/lib/pq" // DB driver
+	"github.com/pkg/errors"
```

Comment:
no need to use this package, use `fmt.Errorf("error description: %w", err)` instead